crates/core/src/pattern/built_in_functions.rs

```diff
@@ -91,12 +89,10 @@ pub struct BuiltInFunction {
 impl BuiltInFunction {
     fn call<'a>(
         &self,
-        args: &'a [Option<Pattern>],
```

Comment:
We have some cloud functions that require the Pattern (not the resolved pattern) so this change won't work for them.